<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_mode</class>
 <widget class="QWidget" name="user_mode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>user_mode</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Breed: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Weight:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="breed_info">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_info">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="age_info">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="weight_info">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="adoptButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Adopt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SkipButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Skip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExportButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Export list</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filter_breed">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Filter by age:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Filter by breed:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filter_weight">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>471</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enable Filtering</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
